genesis/utils/geom.py

```diff
@@ -417,6 +418,67 @@ def normalize(x, eps: float = 1e-9):
         gs.raise_exception(f"the input must be either torch.Tensor or np.ndarray. got: {type(x)=}")
 
 
+def rot6d_to_quat(d6):
+    R = rot6d_to_R(d6)
+    return R_to_quat(R)
+
+
+def quat_to_rot6d(quat):
+    R = quat_to_R(quat)
```

Comment:
Prefer returning directly without intermediary variable for performance concerns